session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -83,6 +83,7 @@
   private ZoneId zoneId;
   private long statementId;
   private int fetchSize;
+  private String rawDataQuery = "raw data query";
```

Comment:
use final to define it as constant, and move the string to other utils class, you may need other query strings later.